docs/how-to-guides/09_how-to-get-table-information.md

```diff
@@ -88,3 +88,66 @@ In the example shown below, the `contract` smart contract's table `profiles` is
   }));
 })();
 ```
+
+## Query Data using the KV API 
```

Comment:
Key-Value API in title
and then the first time when it is used below 
Key-Value API (KV API) .... and from there onwards it can be used as KV API

Reply:
Update to title